x/staking/client/testutil/suite.go

```diff
@@ -61,7 +62,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		val.ValAddress,
 		val2.ValAddress,
 		unbond,
-		fmt.Sprintf("--%s=%d", flags.FlagGas, 202954), //  202954 is the required
+		fmt.Sprintf("--%s=%d", flags.FlagGas, 306313), //  306313 is the required
```

Comment:
wow, gas is increased quite a bit. Is this all due to increased storage requirements? 

it's nbd whatever the reason is, as its not an insane increase and won't be used often

Reply:
It was around 252954. But I  bumped to, 306313  to always pass